src/app.js

```diff
@@ -0,0 +1,57 @@
+import isYes from '../src/isYes.js';
+
+const quizbtn = document.getElementById('quizbtn');
+const correctAnswers = document.getElementById('correct');
+const results = document.getElementById('results');
+
```

Comment:
Your code looks great. I would just add some white space between some lines to separate out code and make it easier to read. an example would be to add a new blank line after 10 and after 20